core/Command/Info/File.php

```diff
@@ -0,0 +1,265 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OC\Core\Command\Info;
+
+use OC\Files\ObjectStore\ObjectStoreStorage;
+use OCA\Circles\MountManager\CircleMount;
+use OCA\Files_External\Config\ExternalMountPoint;
+use OCA\Files_Sharing\SharedMount;
+use OCA\GroupFolders\Mount\GroupMountPoint;
+use OCP\Constants;
+use OCP\Files\Config\IUserMountCache;
+use OCP\Files\FileInfo;
+use OCP\Files\Folder;
+use OCP\Files\IHomeStorage;
+use OCP\Files\IRootFolder;
+use OCP\Files\Mount\IMountPoint;
+use OCP\Files\Node;
+use OCP\Files\NotFoundException;
+use OCP\IL10N;
+use OCP\L10N\IFactory;
+use OCP\Share\IShare;
+use OCP\Util;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class File extends Command {
+	private IRootFolder $rootFolder;
+	private IUserMountCache $userMountCache;
+	private IL10N $l10n;
+
+	public function __construct(IRootFolder $rootFolder, IUserMountCache $userMountCache, IFactory $l10nFactory) {
+		$this->rootFolder = $rootFolder;
+		$this->userMountCache = $userMountCache;
+		$this->l10n = $l10nFactory->get("files");
```

Comment:
```suggestion
		$this->l10n = $l10nFactory->get("core");
```
Since this is in core?